languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,76 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PAS_DE_VIRGULE" name="pas de virgule" default="temp_off">
+          <rule>
+              <pattern>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">et|ou</token>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\2</suggestion>
+              <example correction="et">Elle a acheté des radis<marker>, et</marker> des choux.</example>
+          </rule>
```

Comment:
Tip: the rule editor result becomes more readable if you add the `<exception scope="previous">,</exception>` to the first token ;)

Other than that, some cases might be FPs:

> Pékin**, ou** Beijing, n'a pas toujours été la capitale de l'Empire du Milieu.
> On vu des cafards, des souris et des rats**, et** ce n'est pas tout.